tensor_comprehensions/pybinds/pybind_options.cc

```diff
@@ -55,37 +55,49 @@ PYBIND11_MODULE(mapping_options, m) {
             }
             throw std::runtime_error("Invalid option passed");
           }),
-          "Initialize the mapping options from one of the following:\n 1. naive\n 2. pointwise\n 3. mlp\n 4. conv\n 5. group_conv\n 6. single_thread")
+          "Initialize the mapping options from one of the following:\n"
+          "1. naive\n 2. pointwise\n 3. mlp\n 4. conv\n 5. group_conv\n "
```

Comment:
This drops a space in from of "1.".

Also, I would split the string after every "\n" (here and below).